Content.Client/Cargo/BUI/CargoOrderConsoleBoundUserInterface.cs

```diff
@@ -68,10 +69,20 @@ protected override void Open()
             {
                 if (args.Button.Parent is not CargoProductRow row)
                     return;
+
+                if (row.Description.Text != null)
+                {
+                    var Description = new RichTextLabel();
```

Comment:
this is a local variable which is going unused?

Reply:
You're right, I'll remove it.
I'm new to this project and had to try several ways before I got what I wanted, and I forgot to remove this line before commit.